packages/contracts/contracts/plugin/PluginSetup.sol

```diff
@@ -7,6 +7,7 @@ import {ERC165Checker} from "@openzeppelin/contracts/utils/introspection/ERC165C
 import {ERC1967Proxy} from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
 import {Clones} from "@openzeppelin/contracts/proxy/Clones.sol";
 
+import {createERC1967Proxy as createERC1967} from "../utils/Proxy.sol";
```

Comment:
Let's not alias it, but use it's real name `createERC1967Proxy`.

Reply:
It's aliased because function coming from `utils` shouldn't be the same name that will be in PluginSetup. 

What's your recommendation ?

Reply:
Shouldn't we remove `createERC1967Proxy` from `abstract contract PluginSetup`? 
If `PluginSetup` is used for `PluginClonable` or `Plugin`, the dev might not need it.